.rubocop.yml

```diff
@@ -55,7 +55,12 @@ Style/StringConcatenation:
 # for more discussion
 Style/FetchEnvVar:
   Enabled: false
-  
+
+# Rule enforced after migrating to Ruby 3, to prevent the new syntax which allows for hash values and keyword arguments
+# omission, such as `myMethod(x:, y:)` and `h = { a:, b: }`.
+Style/HashSyntax:
+  EnforcedShorthandSyntax: never
```

Comment:
May I know why adding this rule? I think it'd good to adopt Ruby 3 features from now on?

Reply:
@crazytonyli see paaHJt-5yi-p2

Reply:
Thanks! I have seen that post, but it didn't come to me while reviewing the code. 🙈